### PR TITLE
Shows pending orders in attendees

### LINKED ISSUE #2936: Save attendee data once Pay Now is clicked
Right now the attendee data gets saved only after the order has been placed or completed. What we want to achieve is save the attendee data immediately after pay now is clicked with status="pending".

### PR DESCRIPTION
**Pull request by SaptakS**
Fixes #2936 

screenshot from 2017-01-14 13-32-27

@mariobehling please review.

### CODE REVIEW

File: app/helpers/ticketing.py [hunk 1]
```diff
@@ -274,6 +274,24 @@ def initiate_order_payment(form, paid_via=None):
...
+            for i, firstname in enumerate(holders_firstnames):
+                data = {
+                    'firstname': firstname,
+                    'lastname': holders_lastnames[i]
+                }
+                holder_user = TicketingManager.get_or_create_user_by_email(holders_emails[i], data)
+                ticket_holder = TicketHolder(firstname=data['firstname'],
+                                             lastname=data['lastname'],
+                                             ticket_id=int(holders_ticket_ids[i]),
+                                             email=holder_user.email, order_id=order.id)
```
Comment (rafalkowalski) [inline app/helpers/ticketing.py]: order should be below email

File: app/helpers/ticketing.py [hunk 2]
```diff
@@ -274,6 +274,24 @@ def initiate_order_payment(form, paid_via=None):
...
+                DataManager.add_attendee_role_to_event(holder_user, order.event_id)
+                db.session.add(ticket_holder)
```
Comment (rafalkowalski) [inline app/helpers/ticketing.py]: commit add should be in Datamanager

File: app/views/users/ticket_sales.py [hunk 3]
```diff
@@ -114,7 +114,7 @@ def display_attendees(event_id):
     event = DataGetter.get_event(event_id)
     orders = TicketingManager.get_orders(event_id)
     holders = []
-    for order in orders:
+    for order in orders:
```
Comment (rafalkowalski) [inline app/views/users/ticket_sales.py]: why have you added additional spaces?

File: app/helpers/ticketing.py (hunk 2 above)
Comment (SaptakS) [inline app/helpers/ticketing.py]: Didn't get you.

File: app/views/users/ticket_sales.py (hunk 3 above)
Comment (SaptakS) [inline app/views/users/ticket_sales.py]: Don't know where that came from. Will fix it.

**Pull request merged**